kratos/utilities/apply_function_to_nodes_utility.cpp

```diff
@@ -0,0 +1,97 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
+//  Collaborator:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "utilities/parallel_utilities.h"
+#include "utilities/apply_function_to_nodes_utility.h"
+
+namespace Kratos
+{
+
+void ApplyFunctionToNodesUtility::ApplyFunction(
+    const Variable<double>& rVariable,
+    const double t
+    )
+{    
+    // Get function
+    auto& r_function = *mpFunction;
```

Comment:
same

Reply:
Must be private

Reply:
can you post the compiler error?
I am curious, haven't encountered this

Reply:
is not a compiler error, is an execution error because it must be private